packages/autocomplete-css/completions.json

```
@@ -1,2087 +1,4381 @@
 {
   "tags": [
     "a",
-    "b",
-    "blockquote",
-    "body",
-    "br",
-    "button",
-    "canvas",
-    "code",
-    "div",
-    "em",
-    "form",
-    "footer",
-    "h1",
-    "h2",
-    "h3",
-    "h4",
-    "h5",
-    "h6",
-    "head",
-    "header",
-    "hgroup",
-    "hr",
-    "html",
-    "i",
-    "iframe",
-    "img",
-    "input",
-    "label",
-    "li",
-    "nav",
-    "ol",
-    "p",
-    "pre",
-    "select",
-    "span",
-    "strong",
-    "sub",
-    "summary",
-    "table",
-    "td",
-    "textarea",
-    "th",
-    "title",
-    "tr",
-    "ul",
     "abbr",
+    "acronym",
     "address",
+    "applet",
     "area",
     "article",
     "aside",
     "audio",
+    "b",
     "base",
     "bdi",
     "bdo",
+    "bgsound",
     "big",
+    "blink",
+    "blockquote",
+    "body",
+    "br",
+    "button",
+    "canvas",
     "caption",
+    "center",
     "cite",
+    "code",
     "col",
     "colgroup",
-    "command",
+    "content",
+    "data",
     "datalist",
     "dd",
     "del",
     "details",
     "dfn",
     "dialog",
+    "dir",
+    "div",
     "dl",
     "dt",
+    "em",
     "embed",
     "fieldset",
     "figcaption",
     "figure",
-    "ilayer",
+    "font",
+    "footer",
+    "form",
+    "frame",
+    "frameset",
+    "head",
```

```suggestion
    "frameset",
    "h1",
    "h2",
    "h3",
    "h4",
    "h5",
    "h6",
    "head",
```

these were the only tags removed

---

Thanks for noticing this. Although we can't just add them back into the `completions.json` as that data would then be overridden the next time the update script is used.

It looks like the page from MDN on these tags, located [here](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/Heading_Elements) are actually within a folder called `heading_elements` on their docs repo. Which we use the title of these folders to create the tag list.

So some logic has to be added in to validate all elements added via the tags method like this, and replace things like `heading_elements` with the proper `h1->h6` tags.

It might be worth it to look through the [other folders listed here](https://github.com/mdn/content/tree/main/files/en-us/web/html/element), and see if there's any cases of the same thing happening that we haven't noticed.

---

Alright @icecream17 I've added a small check that allows us to handle this. As well as making it easy to expand in the future if we find this has happened again. Searching through the MDN docs website, I couldn't find any other occurrences of this currently happening.

And I've also of course ran the script again to test the changes and confirm we now get `h1->h6` as well as remove `heading_elements`